src/common/lib/orders.js

```diff
@@ -26,3 +26,11 @@ export async function fetchPaymentDetails(hash) {
 
   return data;
 }
+
+export async function fetchBountyLists(hash) {
+  const { data: { data } } = await axios.get(
+    `${process.env.VUE_APP_DEV_DEBIO_BACKEND_URL}/orders/bounty_list/${hash}`
```

Comment:
what's the difference between `VUE_APP_DEV_DEBIO_BACKEND_URL` and `VUE_APP_BACKEND_API` ?
the `VUE_APP_BACKEND_API` is used in `src/common/lib/ipfs/gcs.js`

Reply:
I think not much 😆, the only difference lies in the `/` here, I don't know what the original purpose was. If you want to change it then I can change it, but I'm afraid it will crash during deployment. Because maybe the env server is currently using the following env.
![image](https://user-images.githubusercontent.com/67890542/143559480-c243b9d3-d2a6-4bd0-b786-201604e44e31.png)

Reply:
this should be handled later in enhance env task that already created in jira. can you please use `VUE_APP_BACKEND_API` instead? the one with `DEV` word will be replaced anyway later

Reply:
sure, I'll update asap